clients/translate/lib/google_api/translate/v2/request_builder.ex

```diff
@@ -177,4 +181,24 @@ defmodule GoogleApi.Translate.V2.RequestBuilder do
   def decode(response, _struct) do
     {:error, response}
   end
+
+  def decode(%Tesla.Env{status: 200, body: body}, struct, dataWrapped: true) do
```

Comment:
Note that this pattern match works for now (as there's only 1 option). In the future, we can refactor if needed.